### PR TITLE
Lighthouse fixes

### PR DESCRIPTION
**Pull request by sssoz**
Accessibility, markup, SEO and some performance-improving fixes (e.g. img optimisation). 

See DOAJ/doajPM#2432

### CODE REVIEW

File: static_content/_site/support/thank-you/index.html [hunk 1]
```diff
@@ -622,10 +622,10 @@ <h2 class="modal__title">Quick search</h2>
               
               
 
-              <h2 class="promo__heading">Plagiarism checking</h2>
+              <h2 class="promo__heading">Provisional ISSNs versus confirmed ISSNs</h2>
```
Comment (Steven-Eardley) [inline static_content/_site/support/thank-you/index.html:625]: I think we don't generate static content before PR anymore, these will often conflict. Generate on merge

**Review by Steven-Eardley — APPROVED** [review]
looks good

**Comment by Steven-Eardley** [comment]
I'm going to merge this as a **feature** branch so it can be tested and validated on `develop` before release. Feature branches should be beased on `develop` and be prefixed `feature/` e.g. `feature/2020-11-26_lighthouse-fixes`

**Comment by sssoz** [comment]
@Steven-Eardley Thank you!

**Pull request merged**